### PR TITLE
[zh-cn] sync version-skew-policy.md and configure-upgrade-etcd.md

### PR DESCRIPTION
**Pull request by asa3311**
sync version-skew-policy.md and configure-upgrade-etcd.md

### CODE REVIEW

File: content/zh-cn/releases/version-skew-policy.md [hunk 1]
```diff
@@ -125,6 +125,48 @@ Example:
...
+
+* `kube-apiserver` 的版本是 **{{< skew currentVersion >}}**
+* `kube-proxy` 支持的版本是 **{{< skew currentVersion >}}**， 
+**{{< skew currentVersionAddMinor -1 >}}**，和 **{{< skew currentVersionAddMinor -2 >}}**
+
+{{< note >}}
+<!--
+If version skew exists between `kube-apiserver` instances in an HA cluster, this narrows the allowed `kube-proxy` versions.
+-->
+如果在 HA 集群中的 `kube-apiserver` 实例之间存在版本偏差，那么这将缩小允许的 `kube-proxy` 版本范围。
```
Comment (tengqm) [inline content/zh-cn/releases/version-skew-policy.md:154]:
```suggestion
如果在 HA 集群中的 `kube-apiserver` 实例之间存在版本偏差，
所允许的 `kube-proxy` 版本范围会被缩小。
```

File: content/zh-cn/releases/version-skew-policy.md [hunk 2]
```diff
@@ -125,6 +125,48 @@ Example:
...
+Example:
+
+* `kube-apiserver` instances are at **{{< skew currentVersion >}}** and **{{< skew currentVersionAddMinor -1 >}}**
+* `kube-proxy` is supported at **{{< skew currentVersionAddMinor -1 >}}**, and **{{< skew currentVersionAddMinor -2 >}}** (**{{< skew currentVersion >}}** is not supported because that would be newer than the `kube-apiserver` instance at version **{{< skew currentVersionAddMinor -1 >}}**)
+-->
+例如：
+
+* `kube-apiserver` 实例的版本是 **{{< skew currentVersion >}}** 和 **{{< skew currentVersionAddMinor -1 >}}**
+* `kube-proxy` 版本为 **{{< skew currentVersionAddMinor -1 >}}** 和 
+**{{< skew currentVersionAddMinor -2 >}}** （**{{< skew currentVersion >}}** 将不被支持，
```
Comment (tengqm) [inline content/zh-cn/releases/version-skew-policy.md:167]:
```suggestion
  **{{< skew currentVersionAddMinor -2 >}}**。（**{{< skew currentVersion >}}** 将不被支持，
```

File: content/zh-cn/releases/version-skew-policy.md [hunk 3]
```diff
@@ -125,6 +125,48 @@ Example:
...
+因为这将比版本为 **{{< skew currentVersionAddMinor -1 >}}** 的 kube-apiserver 实例更新）
```
Comment (tengqm) [inline content/zh-cn/releases/version-skew-policy.md:168]:
```suggestion
  因为该版本将比 **{{< skew currentVersionAddMinor -1 >}}** 的 kube-apiserver 实例更新）
```

File: content/zh-cn/releases/version-skew-policy.md [hunk 4]
```diff
@@ -354,38 +396,31 @@ In-place minor version `kubelet` upgrades are not supported.
...
 ### kube-proxy  {#kube-proxy}
 
-* `kube-proxy` 版本不能比 `kube-apiserver` 版本新。
-* `kube-proxy` 最多只能比 `kube-apiserver` 落后两个次要版本。
-
-例如：
+前提条件：
 
-如果 `kube-proxy` 版本处于 **{{< skew currentVersionAddMinor -2 >}}** 版本：
+kube-proxy 通信的 kube-apiserver 实例的版本是 **{{< skew currentVersion >}}**
```
Comment (tengqm) [inline content/zh-cn/releases/version-skew-policy.md:417]:
```suggestion
* 与 kube-proxy 通信的 kube-apiserver 实例的版本是 **{{< skew currentVersion >}}**。
```

File: content/zh-cn/releases/version-skew-policy.md [hunk 5]
```diff
@@ -354,38 +396,31 @@ In-place minor version `kubelet` upgrades are not supported.
...
+可以选择升级 kube-proxy 实例到 **{{< skew currentVersion >}}** 
+（或者它们可以保持在 **{{< skew currentVersionAddMinor -1 >}}** 或 **{{< skew currentVersionAddMinor -2 >}}**）
```
Comment (tengqm) [inline content/zh-cn/releases/version-skew-policy.md:419]:
```suggestion
  （或者它们可以保持在 **{{< skew currentVersionAddMinor -1 >}}** 或 **{{< skew currentVersionAddMinor -2 >}}**）
```

File: content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 6]
```diff
@@ -702,3 +702,26 @@ Before you start an upgrade, please back up your etcd cluster first.
...
+<!--
+Defragmentation is an expensive operation, so it should be executed as infrequent
+as possible. On the other hand, it's also necessary to make sure any etcd member
+will not run out of the storage quota. The Kubernetes project recommends that when
+you perform defragmentation, you use a tool such as [etcd-defrag](https://github.com/ahrtr/etcd-defrag).
+-->
+碎片整理是一种昂贵的操作，因此应尽可能少地执行此操作。
+另一方面，也有必要确保任何 etcd 成员都不会用尽存储配额。
+Kubernetes 项目建议在执行碎片整理时，
+使用诸如 [etcd-defrag](https://github.com/ahrtr/etcd-defrag)之类的工具。
```
Comment (my-git9) [inline content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md:726]:
```suggestion
使用诸如 [etcd-defrag](https://github.com/ahrtr/etcd-defrag) 之类的工具。
```

File: content/zh-cn/releases/version-skew-policy.md [hunk 7]
```diff
@@ -125,6 +125,49 @@ Example:
...
+Example:
+
+* `kube-apiserver` instances are at **{{< skew currentVersion >}}** and **{{< skew currentVersionAddMinor -1 >}}**
+* `kube-proxy` is supported at **{{< skew currentVersionAddMinor -1 >}}**, and **{{< skew currentVersionAddMinor -2 >}}** (**{{< skew currentVersion >}}** is not supported because that would be newer than the `kube-apiserver` instance at version **{{< skew currentVersionAddMinor -1 >}}**)
+-->
+例如：
+
+* `kube-apiserver` 实例的版本是 **{{< skew currentVersion >}}** 和 **{{< skew currentVersionAddMinor -1 >}}**
+* `kube-proxy` 版本为 **{{< skew currentVersionAddMinor -1 >}}** 和 
+  **{{< skew currentVersionAddMinor -2 >}}**。 （**{{< skew currentVersion >}}** 将不被支持，
```
Comment (my-git9) [inline content/zh-cn/releases/version-skew-policy.md:168]:
```suggestion
  **{{< skew currentVersionAddMinor -2 >}}**。（**{{< skew currentVersion >}}** 将不被支持，
```

**Review by my-git9 — APPROVED** [review]
/lgtm

**Review by my-git9 — APPROVED** [review]
/lgtm

**Pull request merged**